src/pages/Facility/settings/organizations/FacilityOrganizationUsers.tsx

```diff
@@ -115,7 +115,7 @@ export default function FacilityOrganizationUsers({ id, facilityId }: Props) {
                             {userRole.user.first_name} {userRole.user.last_name}
                           </h1>
                           <span className="text-sm text-gray-500">
-                            <span className="mr-2">
+                            <span className="mr-2 md:break-words">
```

Comment:
Let's modify the grid for card container, so that we switch to 2 containers for lg, rather than md.